JSDemos/Demos/PivotGrid/ExcelJsHeaderAndFooter/description.md

```diff
@@ -1,4 +1,4 @@
-The PivotGrid allows you to customize a header and a footer in the exported Excel file. Follow the steps below to configure a header/footer:
+The PivotGrid allows you to customize a header and a footer in the exported Excel file. The <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> library allows you to customize worksheets outside of exported cell regions, and as a result add a header (a title before exported data) and a footer (a note after exported data). Follow the steps below to configure a header/footer:
```

Comment:
The PivotGrid allows you to customize a header and a footer in the exported Excel file. The <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> library allows you to customize worksheets outside of exported cell regions. This, in turns, allows you to add a header (a title before exported data) and a footer (a note after exported data). Follow the steps below to configure a header/footer:
Write Preview